File: PhoneCatalog/Models/Phone.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using MongoRepository;

namespace PhoneCatalog.Models
{
    [CollectionName("phones")]
    public class Phone : Entity
    {
        public string Make { get; set; }
        public string Model { get; set; }
        public string ImageUrl { get; set; }
        public string Date { get; set; }
        public string Type { get; set; }
        public string OS { get; set; }
        public string Processor { get; set; }
        public string Clock { get; set; }
        public string Length { get; set; }
        public string Width { get; set; }
        public string Thickness { get; set; }
        public string Weight { get; set; }
        public string ScreenSize { get; set; }
        public string Resolution { get; set; }
        public string CameraPixelCount { get; set; }
        public string BatteryCapacity { get; set; }
    }
}

File: PhoneCatalog/Scripts/app/factories/phone.js
CatalogApp.factory('Phone', function($resource) {
  var Phone = $resource('/api/phones/:id', { id: '@id' }, { update: { method: 'PUT' } } );
  Phone.prototype.summary = function() {
    return this.OS + ', экран\u00A0' + this.ScreenSize + '\u00A0(' + this.Resolution +
      '), процессор\u00A0' + this.Processor + ', камера\u00A0' + this.CameraPixelCount +
      ', аккумулятор\u00A0' + this.BatteryCapacity;
  };
  return Phone;
});
